generators/app/templates/src/app/shell/__ionic-tabs.shell.component.ts

```diff
@@ -13,31 +13,27 @@ import { HomeComponent } from '@app/home/home.component';
 })
 export class ShellComponent implements OnInit {
   tabs = [
-    { component: HomeComponent, route: 'home', title: 'Home', icon: 'home' },
-    { component: AboutComponent, route: 'about', title: 'About', icon: 'logo-angular' },
-    { component: SettingsComponent, route: 'settings', title: 'Settings', icon: 'cog' }
+    { component: HomeComponent, name: 'home', route: 'home', title: 'Home', icon: 'home' },
+    { component: AboutComponent, name: 'about', route: 'about', title: 'About', icon: 'logo-angular' },
+    { component: SettingsComponent, name: 'settings', route: 'settings', title: 'Settings', icon: 'cog' }
   ];
   selectedTabIndex: number;
   subscription: any;
   constructor(private router: Router,
+              private navCtrl: NavController,
               private activatedRoute: ActivatedRoute) {
   }
   ngOnInit() {
     this.updateTab(this.activatedRoute);
-    // Bind Ionic navigation to Angular router events
-    this.subscription = this.router.events
-      .pipe(filter(event => event instanceof NavigationEnd))
-      .subscribe(() => this.updateTab(this.activatedRoute));
   }
-  onTabChange(selectedTab: Tab) {
-    this.router.navigate([this.tabs[selectedTab.index].route]);
+  onTabChange(selectedTabElm: Tab) {
+    const selectedTab = find(this.tabs, { name: selectedTabElm.name });
```

Comment:
use ES6 array method for `find` instead of lodash, as lodash will be removed from default libs in a coming commit

Reply:
ok will do :) I saw lodash being used below, so I tried to follow.

Reply:
de-lodashed :)